README.md

```diff
@@ -78,4 +78,5 @@ To be able to run you will need to have NodeJS installed locally and follow the
 - On the project folder run `yarn install`.
 - Run `yarn build` to build the React Assets.
 - Run `yarn start`.
-- To build the binaries run `yarn dist {package to create}`.
+- To build the binaries for linux, run `yarn dist {package to create}`.
+- To build the binaries for mac os, run `yarn dist-mac {package to create}`.
```

Comment:
I would prefer Linux, and MacOS (or OSX).

Reply:
Should I change `dist` to `dist-linux` and `dist-mac` to `dist-osx`?
Or leave `dist` as is and only change `dist-mac` to `dist-osx`?

Reply:
Oh I meant in the documentation. Since Linux and macOS are proper nouns, the grammar Nazi in me gets triggered when it sees their capitalisation changed. :P

Reply:
It should be good now